huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go

```diff
@@ -54,80 +59,20 @@ func testAccVPCEndpointApproval_Basic(rName string) string {
 	return fmt.Sprintf(`
 %s
 
-resource "huaweicloud_vpcep_service" "test" {
-  name        = "%s"
-  server_type = "VM"
-  vpc_id      = data.huaweicloud_vpc.myvpc.id
-  port_id     = huaweicloud_compute_instance.ecs.network[0].port
-  approval    = true
-
-  port_mapping {
-    service_port  = 8080
-    terminal_port = 80
-  }
-  tags = {
-    owner = "tf-acc"
-  }
-}
-
-resource "huaweicloud_vpcep_endpoint" "test" {
-  service_id  = huaweicloud_vpcep_service.test.id
-  vpc_id      = data.huaweicloud_vpc.myvpc.id
-  network_id  = data.huaweicloud_vpc_subnet.test.id
-  enable_dns  = true
-
-  tags = {
-    owner = "tf-acc"
-  }
-  lifecycle {
-    ignore_changes = [enable_dns]
-  }
-}
-
 resource "huaweicloud_vpcep_approval" "approval" {
   service_id = huaweicloud_vpcep_service.test.id
   endpoints  = [huaweicloud_vpcep_endpoint.test.id]
 }
-`, testAccVPCEndpoint_Precondition(rName), rName)
+`, testAccVPCEndpoint_Basic(rName))
```

Comment:
In `testAccVPCEndpoint_Basic`,  **approval** is false, then it's not necessary to approval the connection.
I think we should create a endpoint service with approval = true